services/src/main/java/za/co/sfy/lucid/sams/rest/controller/BusRoutesController.java

```diff
@@ -0,0 +1,55 @@
+package za.co.sfy.lucid.sams.rest.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import za.co.sfy.lucid.sams.domain.ServiceStatus;
+import za.co.sfy.lucid.sams.domain.exception.LucidSamsExecutionException;
+import za.co.sfy.lucid.sams.rest.service.BusRoutesService;
+import za.co.sfy.lucid.sams.rest.vo.data.writer.BusRoutesRequest;
+import za.co.sfy.lucid.sams.rest.vo.data.writer.BusRoutesResponse;
+
+import javax.validation.Valid;
+
+/**
+ * @author muzim
+ */
+@RestController
+@RequestMapping("bus-routes")
+public class BusRoutesController {
+
+    private BusRoutesService busRoutesService;
+    private static final Logger logger = LoggerFactory.getLogger(BusRoutesController.class);
+
+    @Autowired
+    public BusRoutesController(final BusRoutesService busRoutesService) {
+        this.busRoutesService = busRoutesService;
+    }
+
+    @PostMapping(
+            produces = MediaType.APPLICATION_JSON_VALUE,
+            consumes = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<BusRoutesResponse> saveBusRoutes(@Valid @RequestBody BusRoutesRequest busRoutesRequest) {
+
+        BusRoutesResponse busRoutesResponse = new BusRoutesResponse();
+
+        try {
+            busRoutesResponse = busRoutesService.saveBusRoutes(busRoutesRequest);
+
+        } catch (LucidSamsExecutionException exception) {
+            logger.warn("Failure occured: " + exception.getMessage(), exception);
```

Comment:
When we catch exceptions and log please user logger.error instead of logger.warning

Reply:
Please make sure to implement this in all controllers